CHANGELOG.md

```diff
@@ -30,6 +30,8 @@
     * Fix `ErrorRateByReadPosition` plotted line names to no longer concatenate multiple read identifiers and no longer have off-by-one read numbering (ex. `Sample1_R2_R3` -> `Sample1_R2`) ([#[1304](https://github.com/ewels/MultiQC/pull/1304))
 * **GATK**
   * Add support for the creation of a "Reported vs Empirical Quality" graph to the Base Recalibration module.
+* **verifyBamID**
+  * Fix "CHIP" value detection to notice when there are "CHIP" values to display. ([#1316](https://github.com/ewels/MultiQC/pull/1316))
```

Comment:
```suggestion
  * Fix "CHIP" value detection to notice when there are "CHIP" values to display. Previously, if any sample does not have a "CHIP" value, no samples with "CHIP" values are displayed.  ([#1316](https://github.com/ewels/MultiQC/pull/1316))
```

Reply:
While your edit is strictly true, it was more generally true the previously NO "CHIP" value would ever be shown even if any sample had "CHIP" data